dev/devicelab/lib/tasks/web_benchmarks.dart

```diff
@@ -99,6 +99,11 @@ Future<TaskResult> runWebBenchmark({ @required bool useCanvasKit }) async {
             profileData.complete(collectedProfiles);
             return Response.notFound('Finished running benchmarks.');
           }
+        } else if (request.requestedUri.path.endsWith('/report-demo')) {
```

Comment:
Let's use a more generic terminology so other benchmarks can use this.

Reply:
`/print-to-console` (see above)

Reply:
`/print-to-console` SGTM

Reply:
Done.